recipes/buttons/callbar_button_with_popover/callbar_button_with_popover_default.story.vue

```diff
@@ -14,6 +14,7 @@
     :button-width-size="buttonWidthSize"
     :text-class="textClass"
     :content-class="contentClass"
+    :open-popover="openPopover"
     @arrow-click="onClick"
```

Comment:
add this:

`@opened="onOpened"`

with this change and the previous on the stories, the `opened` event will appear on the Action section of storybook

<img width="479" alt="image" src="https://github.com/dialpad/dialtone-vue/assets/89984179/390f6db4-d2ee-41a6-89dd-3b51a6f237ac">